tests/clients/test_securitydata.py

```diff
@@ -1313,3 +1320,67 @@ def test_stream_file_by_md5_when_find_file_version_returns_204_status_code_raise
             security_client.stream_file_by_md5("mdhash")
 
         assert e.value.args[0] == PDS_EXCEPTION_MESSAGE.format("mdhash")
+
+    def test_get_all_file_events_calls_search_with_expected_params_when_pg_token_is_not_passed(
```

Comment:
These test signatures need to be renamed as `test_search_all_file_events...()` since the method name changed

Reply:
Changed.